src/Http/Offer/EditOfferRestController.php

```diff
@@ -89,11 +88,7 @@ public function __construct(
 
     public function addLabel(string $cdbid, string $label): Response
     {
-        $this->editService->addLabel(
-            $cdbid,
```

Comment:
Not directly changed and not important in this PR, but do we stick with `cdbid` as a term?

Reply:
No I'd rather just use `id` or `eventId` etc.